sormas-api/src/main/java/de/symeda/sormas/api/vaccination/VaccinationFacade.java

```diff
@@ -40,7 +40,13 @@ VaccinationDto create(
 
 	List<VaccinationDto> getAllVaccinations(String personUuid, Disease disease);
 
+	List<VaccinationListEntryDto> getEntriesList(String personUuid, Disease disease, Integer first, Integer max);
```

Comment:
Add a comment to the create method that explain how it automatically assigns the vaccination to an immunization.